packages/react-router/lib/dom/ssr/components.tsx

```diff
@@ -613,17 +613,24 @@ let isHydrated = false;
 
   @category Types
  */
-export type ScriptsProps = Omit<
-  React.HTMLProps<HTMLScriptElement>,
-  | "children"
-  | "async"
-  | "defer"
-  | "src"
-  | "type"
-  | "noModule"
-  | "dangerouslySetInnerHTML"
-  | "suppressHydrationWarning"
->;
+export interface ScriptsProps
```

Comment:
@pcattori Pinged you in discord but this helps make our component props consistent (and work slightly easier with typedoc).